### ISSUE TITLE
Help Build neural network DOC

### ISSUE THREAD

**Issue by ran-j** [tensorflow/tfjs#678]
#### TensorFlow.js version
    "@tensorflow/tfjs": "^0.12.6",
    "@tensorflow/tfjs-core": "^0.12.15",
    "@tensorflow/tfjs-node": "^0.1.14",

Im building a chatbot with tensorflow JS and NodeJS and I need a light.

Whats is the best acactivation for my layers and optimizer and loss ??

I have the follow:

```
const model = tf.sequential();
  model.add(tf.layers.dense({ units: 256, activation: 'relu', inputShape: [train_x[0].length] }));
  model.add(tf.layers.dropout({rate: 0.2}));
  model.add(tf.layers.dense({ units: 128, activation: 'relu' }));
  model.add(tf.layers.dropout({rate: 0.2}));
  model.add(tf.layers.dense({ units: train_y[0].length, activation: 'softmax' }));
  model.compile({ optimizer: 'adam', loss: tf.losses.meanSquaredError, metrics: ['accuracy'] });

  const xs = tf.tensor(train_x);
  const ys = tf.tensor(train_y);

  //train model
  await model.fit(xs, ys, {
    epochs: 1000,
    batchSize: 8,
    shuffle: true,
    // verbose: 1,
    callbacks: {
      onEpochEnd: async (epoch, log) => {
        console.log(`Epoch ${epoch}: loss = ${log.loss}`);
      }
    }
  })
```

```
_________________________________________________________________
Layer (type)                 Output shape              Param #
=================================================================
dense_Dense1 (Dense)         [null,256]                13056
_________________________________________________________________
dropout_Dropout1 (Dropout)   [null,256]                0
_________________________________________________________________
dense_Dense2 (Dense)         [null,128]                32896
_________________________________________________________________
dropout_Dropout2 (Dropout)   [null,128]                0
_________________________________________________________________
dense_Dense3 (Dense)         [null,10]                 1290
=================================================================
Total params: 47242
Trainable params: 47242
Non-trainable params: 0
_________________________________________________________________
```

I reed the Docs and dont find a good explication for those configs

**Comment by bileschi** [comment]
Hi @ran-j ,  Without knowing much more about your specific data and application, it will be impossible for anyone here to give quality advice.  These parameters are much more art than science.  I recommend you begin with a working existing model such as the sentiment analysis model, and use those defaults while slowly mutating the code to use your data and task.

https://github.com/tensorflow/tfjs-examples/tree/master/sentiment

Best of luck!
